src/lib/dnssd/minimal_mdns/Server.cpp

```diff
@@ -386,7 +386,9 @@ CHIP_ERROR ServerBase::BroadcastImpl(chip::System::PacketBufferHandle && data, u
             }
             else
             {
-                ChipLogError(Discovery, "Attempt to mDNS broadcast failed:  %s", chip::ErrorStr(err));
+                char ifaceName[20];
```

Comment:
There is a constant for interface name max length: `InterfaceId::kMaxIfNameLength`